guides/common/modules/proc_configuring-dns-route53.adoc

```diff
@@ -0,0 +1,20 @@
+[id="configuring_dns_route53_{context}"]
+= Configuring dns_route53
+
+_Route 53_ is a DNS provider by Amazon.
+For more information, see https://aws.amazon.com/route53/[aws.amazon.com/route53].
+
+.Procedure
+. Enable _Route 53_ DNS on your {SmartProxy}:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# {foreman-installer} \
+--enable-foreman-proxy-plugin-dns-route53 \
+--foreman-proxy-dns true \
+--foreman-proxy-dns-provider route53 \
+--foreman-proxy-plugin-dns-route53-aws-access-key _My_AWS_Access_Key_ \
+--foreman-proxy-plugin-dns-route53-aws-secret-key _My_AWS_Secret_Key_
+----
+. In the {ProjectWebUI}, navigate to *Infrastructure* > *{SmartProxies}*.
+. Select your {SmartProxy} and from the *Actions* list, select *Refresh*.
```

Comment:
The installer should take care of the refresh.

Reply:
Thanks, removed.